smriprep/workflows/surfaces.py

```diff
@@ -323,28 +329,44 @@ def init_autorecon_resume_wf(omp_nthreads, name='autorecon_resume_wf'):
         name='outputnode')
 
     autorecon2_vol = pe.Node(
-        fs.ReconAll(directive='autorecon2-volonly', openmp=omp_nthreads),
+        ReconAll(directive='autorecon2-volonly', openmp=omp_nthreads),
         n_procs=omp_nthreads, mem_gb=5, name='autorecon2_vol')
     autorecon2_vol.interface._always_run = True
 
     autorecon_surfs = pe.MapNode(
-        fs.ReconAll(
+        ReconAll(
             directive='autorecon-hemi',
-            flags=['-noparcstats', '-nocortparc2', '-noparcstats2',
-                   '-nocortparc3', '-noparcstats3', '-nopctsurfcon',
-                   '-nohyporelabel', '-noaparc2aseg', '-noapas2aseg',
-                   '-nosegstats', '-nowmparc', '-nobalabels'],
+            flags=['-noparcstats', '-noparcstats2', '-noparcstats3',
+                   '-nohyporelabel', '-nobalabels'],
             openmp=omp_nthreads),
         iterfield='hemi', n_procs=omp_nthreads, mem_gb=5,
         name='autorecon_surfs')
     autorecon_surfs.inputs.hemi = ['lh', 'rh']
     autorecon_surfs.interface._always_run = True
 
-    autorecon3 = pe.MapNode(
-        fs.ReconAll(directive='autorecon3', openmp=omp_nthreads),
+    # -cortribbon is a prerequisite for -parcstats, -parcstats2, -parcstats3
+    cortribbon = pe.Node(ReconAll(directive=Undefined,
+                                  steps=['cortribbon']), name='cortribbon')
```

Comment:
This task motivates the updated interface. If the output of a given step specified in `steps` exists, and its inputs have not been updated, then the step will be skipped. It is analogous to the `directive` input, which is a shorthand for many steps.